.gitignore

```diff
@@ -1,9 +1,7 @@
-.Rbuildignore
 .Rproj.user
 .Rhistory
 .Rdata
 .httr-oauth
 .DS_Store
-data/*
```

Comment:
Removed so everything is there for users when they create a new repo from the template. Notes added in multiple places on README about not exposing sensitive data,